.claude/skills/github/tests/Get-AllPRsWithComments.Tests.ps1

```
@@ -0,0 +1,302 @@
+            $result = Get-AllPRsWithComments -Owner "testowner" -Repo "testrepo" -Since ([datetime]"2025-01-01")
+
+            $result | Should -HaveCount 2
+        }
+
+        It "Respects MaxPages limit" {
+            Mock -ModuleName GitHubCore -CommandName 'Invoke-GhGraphQL' -MockWith {
+                $pr = New-MockPRNode -Number 1 -UpdatedAt "2025-12-15T00:00:00Z"
+                return (New-MockGraphQLData -Nodes @($pr) -HasNextPage $true -EndCursor "cursor_next")
+            }
+
+            $result = Get-AllPRsWithComments -Owner "testowner" -Repo "testrepo" -Since ([datetime]"2025-01-01") -MaxPages 2
```

PSScriptAnalyzer (1.25.0)

[warning] The variable 'result' is assigned but never used.

(PSUseDeclaredVarsMoreThanAssignments)